seahorn/include/assert.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#include <seahorn/seahorn.h>
+/*
+ * Add assert marco to invoke sassert
+ */
+#define assert sassert
```

Comment:
this is dangerous. We only want to check memory safety. Not existing asserts

Reply:
But what if some existing assertions in trusty code are important? I think we cover all pre / post-conditions on verify-c-common.

Reply:
some assertions might be very hard to check and some may require special pre condition. We might decide to do this per file, but not blanket for everything. For now the focus was on memory safety. Should not mix different type of assertions together.

Reply:
Ok. I will remove this commit. Thanks for the clarification.